AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -471,7 +471,8 @@ - (void)_scheduleIvarsForMainDeallocation
 
   // Unwrap the ivar array
   unsigned int count = 0;
-  NSAssert(sscanf(ivarsObj.objCType, "[%u^{objc_ivar}]", &count), @"Unexpected type in NSValue: %s", ivarsObj.objCType);
+  int scanResult = sscanf(ivarsObj.objCType, "[%u^{objc_ivar}]", &count);
+  NSAssert(scanResult, @"Unexpected type in NSValue: %s", ivarsObj.objCType);
```

Comment:
You could be even more specific here and explicitly assert `scanResult == 1`, if you're feeling pedantic. :)